test/e2e-kind.sh

```diff
@@ -41,12 +42,22 @@ docker_exec() {
 create_kind_cluster() {
     echo 'Downloading kind...'
 
-    curl -sSLo "${tmp}/kind" \
+    curl -fsSLo "${tmp}/kind" \
         "https://github.com/kubernetes-sigs/kind/releases/download/$KIND_VERSION/kind-$(uname)-amd64"
     chmod +x "${tmp}/kind"
 
+    curl -fsSLo "${tmp}/entrypoint.sh" "https://gist.githubusercontent.com/d2iq-dispatch/9f67e6a97aafac7f8524dc8d4631ae98/raw/291543d4de29c85f9699c1b11d9c4643cce0f77a/gistfile1.txt"
+    chmod +x "${tmp}/entrypoint.sh"
+
+    cat << EOF > tmp_dockerfile
+FROM kindest/node:$K8S_VERSION
+ADD ./entrypoint.sh /usr/local/bin/entrypoint
+EOF
+
+    docker build -t tmp-dispatch-kind:latest -f tmp_dockerfile "${tmp}"
+
     "${tmp}/kind" create cluster --name "$CLUSTER_NAME" \
-        --config test/kind-config.yaml --image "kindest/node:$K8S_VERSION" \
+        --config test/kind-config.yaml --image "tmp-dispatch-kind:latest" \
```

Comment:
What is this special image `tmp-dispatch-kind` ?

Reply:
@orsenthil it got built two lines above: https://github.com/mesosphere/charts/pull/470/files#diff-37eda1430799e62f92596aa54fae0596R57

Reply:
As soon as our version of entrypoint.sh is contributed to github.com/kubernetes-sigs/kind, we can replace the "gist" url by a link to entrypoint.sh in that repo.

Reply:
Hi @cprovencher : could that be added as a comment with the link to the PR?

Reply:
Done